### PR TITLE
loader to show that the google places api is loading places

### PR DESCRIPTION
**Pull request by DanishWajid**
There is no indicator to show that API is calling. Besides, I have also added the loaderColor, loadingText and styles.loadingText props.

### CODE REVIEW

File: GooglePlacesAutocomplete.js [hunk 1]
```diff
@@ -547,7 +547,7 @@ export const GooglePlacesAutocomplete = forwardRef((props, ref) => {
   };
 
   const _getRowLoader = () => {
-    return <ActivityIndicator animating={true} size='small' />;
+    return <ActivityIndicator animating={true} color={props.loaderColor} size='small' />;
```
Comment (bell-steven) [inline GooglePlacesAutocomplete.js:550]: Let's see if we can add this to the styles.loader prop.

**Review by bell-steven — CHANGES_REQUESTED** [review]
Thanks for the PR.

A few things to keep in mind:
1) we have loader in the `renderRow` function, we probably don't want to have both.
2) Aside from `loadingText`, I really want to avoid adding any more props.
3) How can we make sure this isn't a breaking change, while at the same time improving the activityLoader behavior and making it more customizable at the same time.
4) I you can run yarn lint, I would appreciate it.

**Comment by DanishWajid** [comment]
ok let me fix the issues.

**Comment by DanishWajid** [comment]
I have made some changes.

1- Used the old _renderLoader function made some changes.
2-The three new props (loaderColor, styles.loadingText and  loadingText) are still there because I think it's useful.
3- I change the ` const [loading, setLoader] =useState(false)`  to  `const [loading, setLoading] = useState(false)`.
4- According to my test cases, I don't think it's breaking change. You can double-check.
5- The new code removed in my previous commit is merged again (i have cloned the repo earlier but the changes are merged yesterday that's why it's removed by mistake).

File: GooglePlacesAutocomplete.js [hunk 2]
```diff
@@ -579,8 +590,8 @@ export const GooglePlacesAutocomplete = forwardRef((props, ref) => {
     return rowData.description || rowData.formatted_address || rowData.name;
   };
 
-  const _renderLoader = (rowData) => {
-    if (rowData.isLoading === true) {
+  const _renderLoader = ({ isLoading = false } = {}) => {
```
Comment (bell-steven) [inline GooglePlacesAutocomplete.js:593]: Where are you setting `isLoading?`

**Review by bell-steven — APPROVED** [review]

**Comment by DanishWajid** [comment]
isLoading functionality is implemented previously for _renderRow. I didn't set it just destructured it to avoid code-breaking.

**Pull request merged**